src/Feedback.java

```diff
@@ -0,0 +1,12 @@
+
+
+public class Feedback {
```

Comment:
I love this class. A simple Value Object. It acts as a container of information between other components, and lets them talk in a way meaningful to your domain. 

"I am passing you feedback"

**Suggestion**
If you choose to make these member fields (cow, bull) public to skirt the need for getters - I would expect we define them as `final` so they cannot be changed over time. 

If some feedback changes, it's simply new feedback. 

I set my editor to make everything final by default, and let the IDE remind me when I'm trying to mutate values because I often don't intend to.

In Java you can even set the class to be `final` signalling it cannot be extended.

Reply:
Ah I see yep, will make the fields `final`